frame/ethereum/src/lib.rs

```diff
@@ -250,13 +250,13 @@ impl<T: Trait> Module<T> {
 		ethereum::Transaction,
 		ethereum::Block,
 		TransactionStatus,
-		ethereum::Receipt
+		Vec<ethereum::Receipt>
```

Comment:
I'm not familiar with receipt. Can you add a comment in the code about why this is a vector (basically why there are multiple receipt) ?

Reply:
Sure I will do. For each transaction, there is a receipt counterpart. When you call the `transaction_receipt` rpc method, there are a couple of fields that need to be calculated taking into account the cumulatve receipt data for the block that the transaction you are requesting belongs to (`cumulative_gas_used`, `log_index`).

That's why we need all the receipts in the block.

Reply:
Thanks, can you add that comment also into the code?